lib/tasks/upgrade.rake

```diff
@@ -0,0 +1,56 @@
+# TRANSLATORS: do not translate
+desc <<-END_DESC
+  This task runs predefined upgrade steps.
+
+  Examples:
+    rake upgrade:run
```

Comment:
Should ew namespace this? `foreman:upgrade:run` ?

Reply:
i don't think any other tasks in foreman are  namespaced?  I'm actually in favor of it, it just does not seem like the standard.

Reply:
True, but, never a bad time to start a great trend!